doc/API.md

```diff
@@ -711,253 +711,190 @@ The input byte array should be 80 bytes that form the solved block header. *Unli
-
-#### /renter/load [POST]
-
-Function: Load a .sia file into the renter.
+#### /renter/files
 
-Parameters:
-```
-source string
-```
-'source' is the location on disk of the .sia file being loaded.
+lists the status of all files.
 
-Response:
-```
-struct {
-	filesadded []string
+###### JSON Response [(with comments)](/doc/api/Renter.md#json-response-3)
+```javascript
+{
+  "files": [
+    {
+      "siapath":        "foo/bar.txt",
+      "filesize":       8192, // bytes
+      "available":      true,
+      "renewing":       true,
+      "redundancy":     5,
+      "uploadprogress": 100, // percent
+      "expiration":     60000
+    }
+  ]
 }
 ```
-'filesadded' is an array of renter locations of the files contained in the
-.sia file.
 
+#### /renter/delete/___*siapath___ [POST]
 
-#### /renter/loadascii [POST]
+deletes a renter file entry. Does not delete any downloads or original files,
+only the entry in the renter.
 
-Function: Load a .sia file into the renter.
-
-Parameters:
+###### Path Parameters [(with comments)](/doc/api/Renter.md#path-parameters)
 ```
-asciisia string
+:siapath
 ```
-'asciisia' is the ASCII-encoded .sia file that is being loaded.
 
-Response:
-```
-struct {
-	filesadded []string
-}
-```
-See /renter/load for a description of 'filesadded'
+###### Response
+standard success or error response. See
+[API.md#standard-responses](/doc/API.md#standard-responses).
 
-#### /renter/share [GET]
+#### /renter/download/___*siapath___ [GET]
 
-Function: Create a .sia file that can be shared with other people.
+// TODO: does this call still block until the file has been downloaded?
+downloads a file to the local filesystem.
 
-Parameters:
+###### Path Parameters [(with comments)](/doc/api/Renter.md#path-parameters-1)
 ```
-siapaths    []string
-destination string
+:siapath
 ```
-'siapaths' is an array of the renter paths to be shared. It is comma-delimited.
-
-'destination' is the path of the .sia file to be created. It must end in
-'.sia'.
 
-Response: standard.
-
-#### /renter/shareascii [GET]
-
-Function: Create an ASCII .sia file that can be shared with other people.
-
-Parameters:
+###### Query String Parameters [(with comments)](/doc/api/Renter.md#query-string-parameters-1)
 ```
-siapaths []string
+destination
 ```
-'siapaths' is an array of the nicknames to be shared. It is comma-delimited.
 
-Response:
-```
-struct {
-	asciisia string
-}
-```
-'asciisia' is the ASCII-encoded .sia file.
+###### Response
+standard success or error response. See
+[API.md#standard-responses](/doc/API.md#standard-responses).
 
-#### /renter/delete/{siapath} [POST]
+#### /renter/rename/___*siapath___ [POST]
 
-Function: Deletes a renter file entry. Does not delete any downloads or
-original files, only the entry in the renter.
+renames a file. Does not rename any downloads or source files, only renames the
+entry in the renter.
+// TODO: what happens if newsiapath already exists? What happens if siapath doesn't exist?
```

Comment:
Not sure what the current behavior is, but it makes sense both of these should return errors if they don't already.